doc/articles/getting-started-counter.md

```diff
@@ -0,0 +1,324 @@
+
+Now that we have the image file, we can replace the `TextBlock` with an `Image`. 
+
+- Open the `MainPage.xaml` file in the `Counter` project
+
+- Replace the `TextBlock` with the following `Image` element
+
+    ```xml
+    <Image Width="150"
+            Height="150"
+            Source="Assets/logo.png" />
+    ```
+- The `Width` and `Height` have been set on the `Image` to ensure the image is displayed at the correct size. The `Source` property has been set to the path of the image file inside the `Counter` project.
+
+## Change Layout
+
+The layout of the application uses a `StackPanel` which allows multiple controls to be added as children and will layout them in a vertical stack. An alternative to the `StackPanel` that is often used to control layout within an Uno Platform application is the `Grid`. The `Grid` allows controls to be laid out in rows and columns, and is often used to create more complex layouts.
+
+A `StackPanel` is a good choice for this application as we want the controls to be laid out vertically, one above the other. Let's go ahead and add the remaining controls for the counter.
+
+- Update the `StackPanel` to remove the `HorizontalAlignment` property, as we'll be centering each of the nested elements individually.
+
+    ```xml
+    <StackPanel VerticalAlignment="Center">
+    ```
+
+- Update the `Image` element to center it horizontally and add a margin.
+
+    ```xml
+    <Image Width="150"
+            Height="150"
+            Margin="12"
+            HorizontalAlignment="Center"
+            Source="Assets/logo.png" />
+    ```
+
+- Add a `TextBox` to allow the user to enter the step size
+
+    ```xml
+    <TextBox Margin="12"
+            HorizontalAlignment="Center"
+            PlaceholderText="Step Size"
+            Text="1"
+            TextAlignment="Center" />
+    ```
+
+- Add a `TextBlock` to display the current counter value
+
+    ```xml
+    <TextBlock Margin="12"
+                HorizontalAlignment="Center"
+                TextAlignment="Center"
+                Text="Counter: 1" />
+    ```
+
+- Add a `Button` to increment the counter
+
+    ```xml
+    <Button Margin="12"
+            HorizontalAlignment="Center"
+            Content="Increment Counter by Step Size" />
+    ```
+
+## ViewModel
+
+So far all the elements we've added to the `MainPage` have had their content set directly in the XAML. This is fine for static content, but for dynamic content we need to use data binding. Data binding allows us to connect the UI to the application logic, so that when the application logic changes, the UI is automatically updated.
+
+As part of creating the application, we selected MVVM as the presentation framework. This added a reference to the `CommunityToolkit.Mvvm` package which provides a base class called `ObservableObject` which implements the `INotifyPropertyChanged` interface. This interface is used to notify the UI when a property has changed, so that the UI can be updated.
```

Comment:
```suggestion
As part of creating the application, we selected MVVM as the presentation framework. This added a reference to the [`CommunityToolkit.Mvvm`](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/) package which provides a base class called `ObservableObject` which implements the `INotifyPropertyChanged` interface. This interface is used to notify the UI when a property has changed, so that the UI can be updated.
```